### PR TITLE
Support skipping setting GOPATH for all users

### PR DESCRIPTION
**Pull request by scorphus**
Hi!

Thanks for this useful role!

Maybe more users might be interested in using it like this PR proposes 😄 

Please review and comment. I'm happy making any amendments 😉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ go_version_target: The string that the `go version` command is expected to retur
 
 go_download_location: The full download URL. This variable simply appends the go_tarball variable onto the Go Download URL. This should not need to be modified.
 
+set_go_path: Whether of not set GOPATH for all users. Default is `true`.
```
Comment (jlund) [inline README.md]: Small typo here. Should read "Whether *or* not *to* set GOPATH..."

File: vars/main.yml [hunk 2]
```diff
@@ -1,2 +1,3 @@
 ---
 go_download_location: "https://storage.googleapis.com/golang/{{ go_tarball }}"
+set_go_path: true
```
Comment (jlund) [inline vars/main.yml]: This should probably go in `defaults/main.yml`. Those variables are set at a lower priority level and are therefore easier to override.

Comment (scorphus) [inline vars/main.yml]: That really makes sense! Thanks for pointing that out 😄

File: README.md (hunk 1 above)
Comment (scorphus) [inline README.md]: Oh, didn't notice this on.

**Comment by scorphus** [comment]
This is it. Thank you for reviewing! 😉

**Comment by jlund** [comment]
Thanks for your help!

**Pull request merged**